src/extension/background-script/actions/blocklist/delete.ts

```diff
@@ -0,0 +1,22 @@
+import db from "../../db";
+
+const deleteBlocklist = async (
+  message: { args: { host: string } },
+  sender: unknown
+) => {
+  const host = message.args.host;
+  const blocklist = await db.blocklist
+    .where("host")
+    .equalsIgnoreCase(host)
+    .first();
+
+  if (blocklist?.id) {
+    await db.blocklist.delete(blocklist.id);
+    await db.saveToStorage();
+    return { data: true };
+  } else {
+    return { data: false };
```

Comment:
Should we add a error message here? See i.e.: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/accounts/delete.ts#L35

Reply:
this is the case when there is no blocklist entry, correct?

can't we then simply also return true?

Reply:
agreed. removed data:false, this will always return true. We can revisit if you think there should be a check.